src/Components/Aspire.Milvus.Client/ConfigurationSchema.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Are you missing the assembly level attribute for `ConfigurationSchema` in a AssemblyInfo.cs file?